preflight-checks/package.json

```diff
@@ -31,7 +31,7 @@
     "command-line-usage": "^6.1.1",
     "enquirer": "^2.3.6",
     "execa": "^4.0.0",
-    "node-fetch": "^3.1.1",
+    "node-fetch": "^2.6.7",
```

Comment:
Add to rollback `node-fetch` to the latest version of the previous major version (`2`) due to incompatibility with ES modules.